src/pages/City/City/City.tsx

```diff
@@ -753,7 +764,7 @@ const City = () => {
                 admins.slice(0, adminsToShow).map((admin) => (
                   <Col className="cityMemberItem" key={admin.id} xs={12} sm={8}>
                     <div
-                      onClick={() => canEdit || activeUserRoles.includes(Roles.Supporter) || activeUserRoles.includes(Roles.PlastMember)
+                      onClick={() => userAccesses["EditCity"] || activeUserRoles.includes(Roles.Supporter) || activeUserRoles.includes(Roles.PlastMember)
```

Comment:
if you already check for useAccesses, do you really need to do these checks '|| activeUserRoles.includes(Roles.Supporter) || activeUserRoles.includes(Roles.PlastMember)' ??

Reply:
yeah, because Supporter and PlastMember can't edit